x/arkeo/configs/config_v1_testnet.go

```diff
@@ -0,0 +1,13 @@
+//go:build testnet
+// +build testnet
+
+// For internal testing and mockneting
+package configs
+
+import "github.com/arkeonetwork/arkeo/common"
+
+func init() {
+	int64Overrides = map[ConfigName]int64{
+		MaxSupply: common.Tokens(1_000_000_000),
```

Comment:
What is the benefit here of the override for testnet?

Reply:
the regression tests already have a token supply over 121m (for testing purposes). Setting this override makes it that I don't have to refactor all of those tests

Reply:
Please refactor the tests, this isn't a good pattern where we are writing code to override other values and eventually will cost us more time than we are saving.

Reply:
Having environment overrides is common place on THORChain (see https://gitlab.com/thorchain/thornode/-/blob/develop/constants/constants_testnet.go)

But doing the refactoring work you're talking about would take a lot of time (they all have to be rewritten). Its def not worth the hours of work.